app/src/androidTest/java/ch/epfl/sweng/eventmanager/repository/data/FeedTest.java

```diff
@@ -0,0 +1,73 @@
+package ch.epfl.sweng.eventmanager.repository.data;
+
+import org.json.JSONException;
+import org.json.JSONObject;
+import org.junit.Before;
+import org.junit.Test;
+
+import java.text.ParseException;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+
+public class FeedTest {
+    Feed feed1;
+    Feed feed2;
+    Feed feed3;
+    Feed feed4;
+    Feed feed5;
+    JSONObject obj5;
+
+    @Before
+    public void setUp() throws JSONException {
+        JSONObject obj1 = new JSONObject();
+        obj1.put("created_time", "2018-12-03T16:02:53+0000");
+        obj1.put("message", "event in blue !");
```

Comment:
Maybe you should declared a String object on top instead of rewriting `"event in blue"` or `"message"` everytime